GOVERNANCE.md

```diff
@@ -0,0 +1,126 @@
+Governance for the R data.table project
+
+# Purpose and scope
+
+## This document
+
+The purpose of this document is to define how people related to the project work together, so that the project can expand to handle more contributions, from a larger and more diverse group of contributors.
+
+## The R package
+
+The purpose of the project is to maintain the R data.table package,
+which is guided by the following principles:
+
+* Few (if any) dependencies
+* Time & memory efficiency
+* Concise syntax (minimal redundancy in code)
+* Stable code base (easy for users to upgrade to new data.table, and compatible with old R versions)
+* Comprehensive and accessible documentation
```

Comment:
Still wonder if this and the next bullet should be merged, especially the use of "accessible" which hints at "internationalization" to me. i18n is borne out much more in error/warning space. Possibly:

```suggestion
* Comprehensive and accessible documentation and run-time signals (errors, warnings)
```